src/metalnx-web/src/main/resources/views/collections/fileInfo.html

```diff
@@ -40,7 +40,7 @@
 								title="Permissions">Permissions</span>
 						</a></li>
 
-						<li><a href="#preview" data-toggle="tab" role="tab"
+						<li th:if="${permissionType != 'none'}"><a href="#preview" data-toggle="tab" role="tab"
```

Comment:
same as the other comment... let's keep these to 'positive' matches, so if 'read', 'write', or 'own', rather than 'not none'.

Reply:
@connectbo let's see if we can flip this logic and get this merged.

Reply:
done